duckscript_sdk/src/sdk/std/string/camel_case/help.md

```diff
@@ -0,0 +1,21 @@
+```sh
+var = camel_case text
+```
+
+Converts the provided string into camel case.
```

Comment:
can we add a <br> to the end, just to be sure it breaks the line?

Reply:
done